### ISSUE TITLE
Druid Feral finishers don´t work

### ISSUE THREAD

**Issue by LeRutY** [ottconnect/Legion#88]
Unfortunately still not fixed. Rogues finishers work in the newest version. hansleo21 posted a thread about the combo points issue previously: https://github.com/ottconnect/Legion/issues/42

Is there a way that I can get it working myself? I really want to play as feral druid.

**Comment by ottconnect** [comment]
are you any good in c++ ??

**Comment by LeRutY** [comment]
No, I´m sry. I guess I have to wait, but you did great fixes for other bugs. Please don´t stop working on the repack.

EDIT: Uh, rogues finishers don´t work as well. You lose only one combo point, though you have 2-5 and you´re doing almost no damage. I firstly thought they´re now working correctly. Sorry to repost the same issue. I wanted to have a seperate thread for the combo points.
